src/components/Profile.js

```diff
@@ -70,20 +51,30 @@ class Profile extends Component<ProfileProps, ProfileState> {
       // TODO make below message nice page
       return <div>Member "{this.props.memberId}" does not exist</div>;
     }
-    const fullName = memberData.get('full_name');
-    const youtubeUrl = memberData.get('video_url');
+    const fullName = memberDoc.get('full_name');
+    const youtubeUrl = memberDoc.get('video_url');
     return (
       <div>
         <div className="Green MemberName">{fullName}</div>
         {youtubeUrl && <YoutubeVideo youtubeUrl={youtubeUrl} />}
-        <MemberRelations uid={memberData.id} />
+        <MemberRelations uid={memberDoc.id} mid={memberDoc.get('mid')}/>
       </div>
     );
   }
 }
 
 const Loading = () => {
-  return <div>Loading</div>;
+  return <div>Loading</div>;  // TODO make work
 };
 
-export default Profile;
+function mapStateToProps(state, ownProps) {
+  const isMyProfile = ownProps.match.path === '/me';
+  const memberId = isMyProfile ? null : ownProps.match.params.memberId;
+  const authIsLoaded = state.auth.isLoaded;
+  const authFirebaseUser = state.auth.firebaseUser;
+  const authMemberDoc = getAuthMemberDoc(state);  // TODO will we use this?
```

Comment:
observation after working with the data fetching layer: there's 2 (potentially more) ways of getting the current authorized user: (1) importing the `auth` module, and (2) fetching the user by memberId. Juxtaposed with methods to get another member's doc, can be confusing.

is there a way we could unify the two modules?

Reply:
sounds like a good idea!